android/envoy/src/main/kotlin/org/greatfire/envoy/NetworkIntentService.kt

```diff
@@ -679,6 +684,66 @@ class NetworkIntentService : IntentService("NetworkIntentService") {
         }
     }
 
+    private fun handleSnowflakeSubmit(
+        url: String,
+        captive_portal_url: String,
+        hysteriaCert: String?
+    ) {
+
+        val uri = URI(url)
+        // borrowed from the list Tor Browser uses
+        // https://gitlab.torproject.org/tpo/applications/tor-browser-build/-/merge_requests/617/diffs
+        val ice = "stun:stun.l.google.com:19302,stun:stun.antisip.com:3478,stun:stun.bluesip.net:3478,stun:stun.dus.net:3478,stun:stun.epygi.com:3478,stun:stun.sonetel.com:3478,stun:stun.uls.co.za:3478,stun:stun.voipgate.com:3478,stun:stun.voys.nl:3478"
+        val brokerUrl = ""
+        val front = ""
+        val ampCache = ""
+        val rawQuery = uri.rawQuery
+        val queries = rawQuery.split("&")
+        for (i in 0 until queries.size) {
+            val queryParts = queries[i].split("=")
+            if (queryParts[0].equals("broker")) {
+                brokerUrl = queryParts[1]
+            } else if (queryParts[0].equals("ampCache")) {
+                ampCache = queryParts[1]
+            } else if (queryParts[0].equals("front")) {
+                front = queryParts[1]
+            }
```

Comment:
I forgot to allow overriding the `ice` value

Reply:
does it matter? It's 3 weeks old, I would assume I didn't put that in after the feature was added... I think it was meant to be a reminder